frontend/src/components/Banner.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div>
+    <div v-if="!hidden" class="banner">
+        <span class="text">{{text}}</span>
+        <a :href="link" class="link" target="_blank" rel="noopener noreferrer">
+          <button class="button" @click="hidden=true">{{linkText}}</button>
+        </a>
+        <svg @click="hidden=true" class="closeBtn" width="27.667" height="27.667" viewBox="0 0 27.667 27.667">
```

Comment:
I'm not sure what are the requirements in this task, but are we sure we want it always showing, even after user dismisses it (on component rerender, so, coming to the site for the second time), seems a bit aggressive. We could store the hidden in localStorage and save that user dismissed it, but as I mentioned before, I do not know what the requirements here are